lib/DependencyScan/ModuleDependencyScanner.cpp

```diff
@@ -862,21 +904,24 @@ ModuleDependencyScanner::resolveAllClangModuleDependencies(
   return;
 }
 
-void
-ModuleDependencyScanner::resolveHeaderDependencies(
+void ModuleDependencyScanner::resolveHeaderDependencies(
+    const ModuleDependencyID &rootModuleID,
     ArrayRef<ModuleDependencyID> allSwiftModules,
     ModuleDependenciesCache &cache,
     ModuleDependencyIDSetVector &allDiscoveredHeaderDependencyClangModules) {
   for (const auto &moduleID : allSwiftModules) {
     auto moduleDependencyInfo = cache.findKnownDependency(moduleID);
     if (!moduleDependencyInfo.getHeaderClangDependencies().empty()) {
-      allDiscoveredHeaderDependencyClangModules.insert(moduleDependencyInfo.getImportedSwiftDependencies().begin(),
-                                                       moduleDependencyInfo.getImportedSwiftDependencies().end());
+      allDiscoveredHeaderDependencyClangModules.insert(
+          moduleDependencyInfo.getImportedSwiftDependencies().begin(),
```

Comment:
I know this wasn't introduced in this PR, but I think this should be `getHeaderClangDependencies().begin()`.